src/sentry/identity/services/identity/model.py

```diff
@@ -21,6 +21,7 @@ class RpcIdentity(RpcModel):
     idp_id: int  # IdentityProvider id
     user_id: int
     external_id: str
+    scopes: list[str]
```

Comment:
This needs to be optional to start I think. Otherwise it can break serialization